crates/bevy_remote/src/schemas/json_schema.rs

```diff
@@ -538,6 +616,32 @@ mod tests {
             "a"
           ]
         });
-        assert_eq!(schema_as_value, value);
+        assert_normalized_values(schema_as_value, value);
+    }
+
+    fn assert_normalized_values(one: Value, two: Value) {
```

Comment:
Nit: Could you add a doc comment for why this is needed? That way contributors don't have to try and figure it out for themselves.